packages/create-ima-app/examples/feed/mock/MockHttpAgent.js

```diff
@@ -1,4 +1,4 @@
-import { IMAError, HttpAgent } from '@ima/core';
+import { GenericError as IMAError, HttpAgent } from '@ima/core';
```

Comment:
Mohli by jsme misto prejmenovani GenericError na IMAError proste pouzit GenericError? Prijde mi to takhle zbytecne, kdyz to s nicim nekonflikti a jelikoz je to takhle v examplech, tak naznacujeme, ze se to takhle ma pouzivat.